python/ray/data/datasource/parquet_datasource.py

```diff
@@ -47,6 +47,15 @@
 FRAGMENTS_PER_META_FETCH = 6
 PARALLELIZE_META_FETCH_THRESHOLD = 24
 
+# The `num_cpus` for each metadata prefetching task.
+# Default to 0.5 instead of 1 because it is cheaper than normal read task.
+NUM_CPUS_FOR_META_FETCH_TASK = 0.5
```

Comment:
so currently, there is no configurable way (e.g. through `DataContext`) to modify this right?

Reply:
yes, I am hesitating to add to `DataContext` (which exposed to users). If we see this is a value needed by users, we can expose it later. For now, let's start w/ a varilable, so at least we (internal developer) can change it.